lib/cylc/__init__.py

```diff
@@ -37,7 +37,7 @@ def environ_init(argv0=None):
         if cylc_dir != os.getenv('CYLC_DIR', ''):
             os.environ['CYLC_DIR'] = cylc_dir
 
-        cylc_dir_lib = os.path.join(cylc_dir, 'lib')
+        cylc_dir_lib = os.path.normpath(os.path.join(cylc_dir, 'lib'))
```

Comment:
Yes and no, `normpath` is really just a string manipulation, so should be quick. `realpath` requires access to the file system and so may slow things down. For our purpose here, I think `normpath` is enough.